target/linux/generic/hack-5.4/710-net-dsa-mt7530-disable-eee-on-ports.patch

```diff
@@ -0,0 +1,52 @@
+From d422285f34bc95b579e03f29853165a976182858 Mon Sep 17 00:00:00 2001
+From: Chen Minqiang <ptpt52@gmail.com>
+Date: Wed, 8 Apr 2020 10:39:39 +0800
+Subject: [PATCH] net: dsa: mt7530: disable eee on ports
+
+Signed-off-by: Chen Minqiang <ptpt52@gmail.com>
+---
+ drivers/net/dsa/mt7530.c | 21 +++++++++++++++++++++
+ 1 file changed, 21 insertions(+)
+
+diff --git a/drivers/net/dsa/mt7530.c b/drivers/net/dsa/mt7530.c
+index eb4f651b8dcc..4359fb2a3c1c 100644
+--- a/drivers/net/dsa/mt7530.c
++++ b/drivers/net/dsa/mt7530.c
+@@ -1295,6 +1295,22 @@ mt7530_set_jumbo(struct mt7530_priv *priv, u8 kilobytes)
+ 		mt7530_rmw(priv, MT7530_GMACCR, MAX_RX_PKT_LEN_MASK, MAX_RX_PKT_LEN_1536);
+ }
+ 
++static void
++mt7530_port_disable_eee(struct dsa_switch *ds, int i)
++{
++	/* Disable EEE */
++	mt7530_phy_write(ds, i, 13, 0x7);
++	mt7530_phy_write(ds, i, 14, 0x3c);
++	mt7530_phy_write(ds, i, 13, 0x4007);
++	mt7530_phy_write(ds, i, 14, 0x0);
++
++	/* Disable EEE 10Base-Te */
++	mt7530_phy_write(ds, i, 13, 0x1f);
++	mt7530_phy_write(ds, i, 14, 0x027b);
++	mt7530_phy_write(ds, i, 13, 0x401f);
++	mt7530_phy_write(ds, i, 14, 0x1177);
```

Comment:
Please take a look at mt753x driver under mediatek and replace these magic values with a mmd_write.